### PR TITLE
Module 3 Week 3: Adding Style for Book Store App

### PR DESCRIPTION
**Pull request by usamamaliq**
In this pull request, the styling for the book store application was completed and the remaining elements were added.

Regards,
Usama Masood

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+  text-decoration: none;
+}
+
+.BooksContainer {
+  padding: 4%;
+}
+
+.bookMainContainer {
+  display: flex;
+  align-items: center;
```
Comment (tufoinnkuo10) [inline src/App.css:51]: - [x] nice work. design is as expected. please kindly add the the user profile place holder like in the original design. nice work
yours
Screenshot (697)_LI
zeplin
Screenshot (696)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @usamamaliq 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Git-flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.BookContainer {
+  width: 40vw;
+}
+
+.percentContainer {
+  display: flex;
```
Comment (Gambit142) [inline src/App.css:65]: - [OPTIONAL] One good CSS practice is not to repeat the same rules for multiple **_selectors/elements_**. In your `src/App.css` file, you repeated a lot of **_declarations/rules_** for multiple selectors. To fix this, you should create a class for the repeated declaration and use the class on the target element. For example

#### In your CSS file you could do the following:

```CSS
.display-flex {
  display: flex;
}
```

#### and then in your _**JSX**_ file you could do the following:
```HTML
<div className="display-flex">
  ...
</div>
```

Kindly implement this concept for the following repeated rules:

- `display: flex;`
- `border: solid 1px #e8e8e8;`
- `font-family: Montserrat, sans-serif;`
- `font-family: Roboto Slab, sans-serif;`
- `color: #121212;` and other rules that you repeated. 👍 

#### Hint: You can group some of them into a class and use them on the desired elements that you know need the same declarations on the webpage. 👍

**Review by Gambit142 — APPROVED** [review]
Hi @usamamaliq 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Descriptive pull request title ✔️ 
- Descriptive pull request summary ✔️ 

### Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

- Kindly consider the optional suggestions I made in the comments under the review 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**